src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -31,6 +31,8 @@ public class InventoryMock implements Inventory
 	private final InventoryHolder holder;
 	private final InventoryType type;
 
+	private int maxStackSize = 64;
```

Comment:
There should be a constant that this is initialized to rather than directly to 64.

Reply:
Fixed